rocketpy/rocket/rocket.py

```diff
@@ -1170,6 +1166,29 @@ def add_parachute(
         self.parachutes.append(parachute)
         return self.parachutes[-1]
 
+    def add_sensor(self, sensor, position, x_position=0, y_position=0):
+        """Adds a sensor to the rocket.
+
+        Parameters
+        ----------
+        sensor : Sensor
+            Sensor to be added to the rocket.
+        position : int, float, tuple
+            Position, in meters, of the sensor's coordinate system origin
+            relative to the user defined rocket coordinate system.
+        x_position : int, float, optional
+            Distance in meters by which the CM is to be translated in the x
+            direction relative to geometrical center line. Default is 0.
+        y_position : int, float, optional
+            Distance in meters by which the CM is to be translated in the y
+            direction relative to geometrical center line. Default is 0.
+
+        Returns
+        -------
+        None
+        """
+        self.sensors.add(sensor, Vector([x_position, y_position, position]))
```

Comment:
It will be a problem. The sensor readings are saved in itself, so the final saved measurements would be a mix of the measurements in different positions.

I think this is something that should be changed. I'll work on it